### PR TITLE
Feature/vertical image transition

### PR DESCRIPTION
**Pull request by LefBoutham**
## Description
- Add `slideVertically` prop to enable vertical sliding

## Changes
- Add new `slideVertically` boolean prop
- Add new chevron icons and appropriate styles for top / bottom slide buttons
- Add vertical swiping logic for touch-screen devices
- Add checkbox for testing vertical scrolling for example app
- Tweak `translate` logic to enable vertical sliding

## Testing
- Review code changes thoroughly
- Manual test vertical scrolling using the example app for desktop and mobile devices

### CODE REVIEW

**Comment by xiaolin** [comment]
Hey, thanks for the PR. I moved some files around as I was working on a refactor locally. 

As for why its not updating I'm guessing you may need to add something in `componentDidUpdate` when `slideVertically` goes from `false` to `true`

File: src/ImageGallery.js [hunk 1]
```diff
@@ -49,6 +51,7 @@ class ImageGallery extends React.Component {
       isFullscreen: false,
       isSwipingThumbnail: false,
       isPlaying: false,
+      slideVertically: props.slideVertically,
```
Comment (xiaolin) [inline src/ImageGallery.js:54]: `slideVertically ` looks like it can just be a prop and does not need to be in `state` as its never changed.

Comment (LefBoutham) [reply src/ImageGallery.js:54]: Got it, thanks for the feedback. I'll have time to finalize and refactor within the next two weeks 👍

**Comment by LefBoutham** [comment]
@xiaolin tweaks and conflicts addressed, example app working now as intended

**Comment by LefBoutham** [comment]
🏓 @xiaolin

**Pull request merged**